core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeExportService.java

```diff
@@ -53,17 +53,21 @@ public EnrolleeExportService(ProfileService profileService,
         this.objectMapper = objectMapper;
     }
 
-    public void export(ExportOptions exportOptions, UUID portalId, UUID studyEnvironmentId, OutputStream os) throws Exception {
-        List<ModuleExportInfo> moduleExportInfos = generateModuleInfos(exportOptions, portalId, studyEnvironmentId);
-        var enrolleeMaps = generateExportMaps(portalId, studyEnvironmentId,
+    /**
+     * exports the specified number of enrollees from the given environment
+     * The enrollees will be returned most-recent first
```

Comment:
Most recently enrolled? Or most recently updated? Or something else?

Reply:
good question, updated comment to clarify it's most-recently created